new/PRS13969GabraZaraSyon.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13969GabraZaraSyon" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Owner</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key=""/>
```

Comment:
key="SH" ( I will try to check why this was not done by the app)

Reply:
the initials are also missing in the `who `attribute of the `change `element

NB this is the reason the committed record was red in Oxygen, it should be green